### ISSUE TITLE
A11y_code.visualstudio.com_Docs_Left right Navigation_Screenreader: Screenreader announces actionable links as read only. 

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3701]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.3.1;#Win10;#Visual Studio Code;#Narrator;#CAITestPass_May2020;#FTP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)
Screen reader: Narrator, NVDA Version 2020.1

**Repro Steps**:
1. Launch the application URL :https://code.visualstudio.com
2. Navigate to "Docs" link.
3. Left and right navigation will appear.
4. Turn on screen reader and navigate to left navigation content using CapsLock+RightArrow Key.

**Actual**:
Screen reader is announcing actionable Links in left navigation as read only. 

This issue is reproducing only in ChromiumEdge+Narrator.
In Chrome+NVDA screen reader is not announcing it as read only.
 
**Expected**:
Screen reader should not announce actionable controls as read only.  

**Attachment link for Reference**:
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/ESqE3-yUC-hLni-hhJ0C6_kBfglz67QQmbHrw4sHtFvs6g?e=gSKqmc

**Comment by JacksonKearl** [comment]
Unable to reproduce. Tried with full verbosity settings in Narrarator, NVDA, and Voiceover.

**Comment by Rama8345** [comment]
**Github Tags**:#Closed;
